### ISSUE TITLE
Generate or create "mock" local costmap containing projected 2D footprints of construction objects

### ISSUE THREAD

**Issue by nlitz88** [nlitz88/workzone#6]
This task is **essential** for work on the two stages of this pipeline to be done in parallel.

Essentially, the first stage of the system is supposed to produce a 2D costmap with detected construction objects scattered around it. Then, the second stage is supposed to take that 2D array (basically a 2D image or 2D array) and _somehow_ identify groups of construction objects and draw a line around each grouping to define the boundary (which denotes "non-drivable" area.

In order to not hold back those working on the second stage, we **NEED** a "fake" costmap to be made (perhaps by hand in something like photoshop, mspaint, or that weird photo editor available on Ubuntu @CMUBOB97 you know what I'm talking about) so that work can be started on the "grouping" stage algorithm (or whatever approach we end up going with). This 2D grid/costmap will essentially be the interface between the two stages for now, so the sooner we create a mock output for that second stage to work on, the better.

Down below, whenever you get some time to think about it, can you think of any requirements we need to have on these mock costmaps? Also, maybe it'd be good to look at some sample outputs from algorithms like [BEVFusion](https://github.com/mit-han-lab/bevfusion) so that our mock costmap is somewhat representative of what we will really see. Hell, even a screenshot of one of their sample outputs could work pretty well.

**Comment by nlitz88** [comment]
Made a folder here where we can put test costmap files: https://drive.google.com/drive/folders/1m4G2hZEJ587Vx46FrZNi-5IeETCRe3E6?usp=drive_link

Could maybe start with a python script to just read in one of these files and then see what you can do with that. It could be as simple as literally just loading in one of these images--whatever works!

Update this issue with progress and we can add a new task/issue for prototyping different grouping approaches and how we will parse the received costmap (I.e., will we use an adjacency list to maintain the construction objects found in the map, etc?)

**Comment by nlitz88** [comment]
![Image](https://github.com/nlitz88/workzone/assets/9357423/f270d9c0-91da-47b0-bdfc-e3f8ac1f206c)

Divinci
